### PR TITLE
Feature/111 numbers rating

### PR DESCRIPTION
**Pull request by megarocks**

### CODE REVIEW

**Comment by megarocks** [comment]
https://github.com/Abolitionist-Project/QuantiModo-WordPress-Plugin/issues/111

File: css/qmwp.css [hunk 1]
```diff
@@ -595,4 +595,24 @@ body.qmwp-lightbox-visible {
...
+    padding: 10px 0;
+    border-width: 1px 0;
+    border-color: #000;
+    border-style: solid;
+}
+
+table.available-shortcodes tr:last-of-type td {
+    border: none;
+}
+
```
Comment (mikepsinn) [inline css/qmwp.css]: Codacy Issue found: [Element (td.shortcode) is overqualified, just use .shortcode without element name.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1049789523/issues/source?bid=2836188&fileBranchId=2883147#l615)

File: js/qm-numbers-rating.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            });
+
+        } else {
+            console.error(variable);
+            jQuery('.shortcode-content').hide();
+            alert('Variable: ' + qmwpShortCodeDefinedVariable + '\n can not be tracked with this shortcode.');
+        }
+    });
+
+    var ratingButtonClicked = function (value, variable) {
```
Comment (mikepsinn) [inline js/qm-numbers-rating.js]: Codacy Issue found: ['ratingButtonClicked' is defined but never used.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1049789498/issues/source?bid=2836188&fileBranchId=2883147#l46)

File: js/qm-numbers-rating.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        if (qmShortCodeDefinedNegative === 'true') {
+            jQuery(button).addClass(colorClasses[colorClasses.length - i - 1]);
+        } else {
+            jQuery(button).addClass(colorClasses[i]);
+        }
+
+    }
+
+    Quantimodo.getVariableByName(qmShortCodeDefinedVariable, function (variable) {
+        if (variable && variable.abbreviatedUnitName == '/5') {
```
Comment (mikepsinn) [inline js/qm-numbers-rating.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1049789498/issues/source?bid=2836188&fileBranchId=2883147#l29)

File: js/qm-numbers-rating.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+            combinationOperation: variable.combinationOperation,
+            unit: variable.abbreviatedUnitName
+        }];
+
+        Quantimodo.postMeasurementsV2(payload, function (response) {
+
+            var resultHolder = jQuery('#result-string');
+            resultHolder.html('');
+
+            if (response.status == 201) {
```
Comment (mikepsinn) [inline js/qm-numbers-rating.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1049789498/issues/source?bid=2836188&fileBranchId=2883147#l71)

File: QMWP.php [hunk 5]
```diff
@@ -1355,21 +1357,27 @@ function qmwp_bargraph_scatterplot_timeline($attributes)
...
 
         $version = $attributes['version'];
 
         $pluginContentHTML = $this->get_plugin_template_html('qmwp-bargraph-scatterplot-timeline', $version);
 
-        $variable = $attributes['variable'];
+        $variable = $attributes['examined_variable_name'];
+
+        if ($attributes['show_predictors_or_outcomes'] == 'outcomes') {
+            $showPredictorsOrOutcomes = 'cause';
```
Comment (mikepsinn) [inline QMWP.php]: Codacy Issue found: [Avoid excessively long variable names like $showPredictorsOrOutcomes. Keep variable name length under 20.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1049789528/issues/source?bid=2836188&fileBranchId=2883147#l1371)

**Pull request merged**